addons/wowl/static/src/services/rpc.ts

```diff
@@ -82,6 +85,7 @@ function jsonrpc(env: OdooEnv, url: string, params: Params): Promise<any> {
 export const rpcService: Service<RPC> = {
   name: "rpc",
   deploy(env: OdooEnv): RPC {
+    let rpcId: number = 0;
```

Comment:
I would have seen the rpc counter more like a static thing.
Just wondering if there are any possibilities that we would have differents RPC services running at the same time ?

Reply:
Having the rpcId defined here, it will be reset in each test, which is nice.